server/routes/alerts.ts

```diff
@@ -0,0 +1,70 @@
+import { type RequestHandler, Router } from 'express'
+import asyncMiddleware from '../middleware/asyncMiddleware'
+import type { Services } from '../services'
+import logger from '../../logger'
+import { getAlertName, getAlertType } from '../utils/utils'
+
+interface AlertListResponseDataItem {
```

Comment:
Should really go in `/server/@types/index.d.ts` and imported so it can be used elsewhere if need be, or quite probably imported from the `strapi-api.d.ts` via `/server/data/strapiApiTypes.ts` ?